documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -1,206 +1,58 @@
 # Install the Clear Containers runtime on RHEL 7.3
 
+## Requirements
 
-## Introduction
+In order to run cc-oci-runtime, you will need Docker. Currently cc-oci-runtime supports Docker v1.12.1 to v1.13.
```

Comment:
@GabyCT - I thought this (and other occurrences) had been changed to `cc-oci-runtime`?

@rcaballeromx - I can't see mention of it in the comments, but have you suggested maybe that this should not be formatted in fixed-width font (as we're using it as a project name?), or is this an oversight?

(I've raised https://github.com/01org/cc-oci-runtime/issues/788 so we can get clarity on this).